azure-functions-extension-fastapi/setup.py

```diff
@@ -0,0 +1,41 @@
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License.
+
+from setuptools import setup, find_packages
+
+EXTRA_REQUIRES = {
+    'dev': [
+        'flake8~=4.0.1',
+        'flake8-logging-format',
+        'mypy',
+        'pytest',
+        'pytest-cov',
+        'requests==2.*',
+        'coverage',
+        'pytest-instafail'
+    ]
+}
+
+setup(
+    name='azure-functions-extension-fastapi',
+    version='0.0.1',
+    author='Azure Functions team at Microsoft Corp.',
+    author_email='azurefunctions@microsoft.com',
+    description='FastApi Python worker extension for Azure Functions.',
+    packages=find_packages(exclude=[
+        'azure.functions.extension', 'azure.functions',
+        'azure', 'tests', 'samples'
+    ]),
+    classifiers=[
```

Comment:
Also add the same classifiers as in the base and blob extension setup.pys?

Reply:
updated